apps/prairielearn/src/pages/instructorAssessmentRegrading/instructorAssessmentRegrading.html.ts

```diff
@@ -0,0 +1,159 @@
+                      To regrade assessment instances for specific students, use the "Action"
+                      dropdown on the
+                      <a
+                        href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                          .id}/instances"
+                        >assessment instances page </a
+                      >.
+                    </p>
+                  </div>
+                `
+              : ''}
+
+            <div class="table-responsive">
+              <table class="table table-sm table-hover">
+                <thead>
+                  <tr>
+                    <th>Number</th>
+                    <th>Date</th>
+                    <th>Description</th>
+                    <th>User</th>
+                    <th>Status</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${regradingJobSequences && regradingJobSequences.length > 0
+                    ? regradingJobSequences.map((jobSequence) => {
+                        return html`
+                          <tr>
+                            <td>${jobSequence.job_sequence.number}</td>
+                            <td>${jobSequence.start_date_formatted}</td>
+                            <td>${jobSequence.job_sequence.description}</td>
+                            <td>${jobSequence.user_uid}</td>
+                            <td>
+                              ${renderEjs(__filename, "<%- include('../partials/jobStatus'); %>", {
+                                status: jobSequence.job_sequence.status,
+                              })}
+                            </td>
+                            <td>
+                              <a
+                                href="${resLocals.urlPrefix}/jobSequence/${jobSequence.job_sequence
+                                  .id}"
+                                class="btn btn-xs btn-info"
+                                >Details</a
+                              >
+                            </td>
+                          </tr>
+                        `;
+                      })
+                    : html`
+                        <tr>
+                          <td colspan="6">No previous regradings.</td>
+                        </tr>
+                      `}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function regradeAllAssessmentInstancesModal({
+  assessmentSetName,
+  assessmentNumber,
+  csrfToken,
+}: {
+  assessmentSetName: string;
+  assessmentNumber: string;
+  csrfToken: string;
+}) {
+  return Modal({
+    id: 'regrade-all-form',
+    title: 'Regrade all assessment instances',
+    body: html`
+      Are you sure you want to regrade all assessment instances for
+      <strong> ${assessmentSetName} ${assessmentNumber} </strong>
+      ? This cannot be undone.
```

Comment:
Note that this introduces a space between the number and the question mark, which looks weird.
```suggestion
      <strong>${assessmentSetName} ${assessmentNumber}</strong>?
      This cannot be undone.
```